_includes/header.html

```diff
@@ -1,7 +1,7 @@
 <header id="top" class="header">
   <div class="text-vertical-center">
-    <h1>Infinite Activities</h1>
-    <h3>Activities to get people talking</h3>
+    <h1>class="header-text"Infinite Activities</h1>
```

Comment:
In needs to be like this 

    <h1 class="header-text">Infinite Activities</h1>